includes/Stories_Renderer/Renderer.php

```diff
@@ -239,6 +247,16 @@ public function assets() {
 			[],
 			WEBSTORIES_VERSION
 		);
+
+		if ( ! $this->is_amp_request() ) {
+
+			$this->enqueue_style( Embed_Base::STORY_PLAYER_HANDLE );
+			$this->enqueue_script( Embed_Base::STORY_PLAYER_HANDLE );
+
+			// Web Stories Lightbox script.
+			$this->enqueue_script( 'lightbox', [ Embed_Base::STORY_PLAYER_HANDLE ] );
```

Comment:
Maybe this string could be a const on the class. Keep it inline with `STORY_PLAYER_HANDLE`